base-theme/layouts/partials/external-link.html

```diff
@@ -0,0 +1,13 @@
+{{- $text := default .title .text | htmlEscape | .context.RenderString -}}
+{{- $isImage := default false .is_image -}}
+{{- $href := default "" .external_url -}}
+
+{{- if and .is_broken .backup_url -}}
+  {{- $href = .backup_url -}}
+{{- end -}}
+
+{{- if $isImage -}}
+  <img src="{{- $href -}}" alt="{{- $text -}}" />
```

Comment:
So, for resources we have `resource.html` and `resource_link.html`. The `resource.html` shortcode should really probably be called `resource_embed.html`, as its purpose is to take `resource` type content and embed it directly inline, whether that be a video, image or PDF. This is the difference between the "Add Link" and "Embed Resource" buttons in `ocw-studio`. The former will use the `resource_link.html` shortcode, and the latter will use `resource.html`. For images at least, I feel like we should differentiate between the shortcodes used to embed vs generate a link. Users might want to embed an external image, but they also might want to simply post a link to it. Embedding an external image also seems kind of risky, as the content of that image could change out from under us and either break or be a different image than the author intended. Whether or not to allow embedding of external images is a separate discussion, but I think we should at least create separate shortcodes for embedding external links vs simply creating `a` tags for them.

Reply:
Thank you for explaining the difference. It makes sense.

External image embed was added for legacy resources. In [this recent review](https://github.com/mitodl/ocw-hugo-projects/pull/271#discussion_r1466755861) we've decided to not support image embeds at all.